docs/decisions/0006-transmit-updates-only.rst

```diff
@@ -0,0 +1,73 @@
+Transmitting Only Updates IDA
+-----------------------------
+
+Status
+======
+
+Draft
+
+Context
+=======
+The transmit_content_metadata jenkins job (along with the underlying integrated_channels.transmit_content_metadata()
+task and export() method) makes an uncomfortably large number of calls to the the enterprise-catalog service’s
+/api/v1/enterprise-catalog/<uuid>/get_content_metadata endpoint with a relatively low number of customers which results
+in an unacceptably high database load.
+
+usage and causing a myriad of alerts and warnings.
```

Comment:
Is this supposed to be a continuation of the previous sentence?

Reply:
I thiiiink I just meant to delete it haha